### DISCUSSION TITLE
在看图中新增不带平滑的查看模式

### DISCUSSION THREAD

**Discussion by fishBone000** [linuxdeepin/developer-center#4992] [Features Request & Ideas | 特性请求 & 头脑风暴]
现在的看图对低分辨率图像放大后，会自动对图像进行平滑渲染，但是如果要看的图像是像素风的图片的话，因为这个平滑图片就直接糊掉了
比如Minecraft的16x16材质
当然可以用gimp之类的查看，但是那样很麻烦

**Comment by BLumia** [comment]
cc @meiyixiang 

顺便安利下 https://github.com/BLumia/pineapple-pictures ，放大时位图会原样放大，不做抗锯齿。其实我也不知道为什么别的图片查看器都做这个抗锯齿... 😂
